staging/com.scoreSmart/dataset.tt

```diff
@@ -0,0 +1,10 @@
+dataset @com.scoreSmart language "en" {
```

Comment:
Is this actually using the ScoreSmart app? It looks like the domain of that is scoresmart.ro ?

Also, typically we use everything lowercase (or snake_case) for function and class names in ThingTalk (unlike JS, which is camelCase)